shared/ssz/docs/SimpleSerialize.md

```diff
@@ -0,0 +1,121 @@
+# Simple Serialize (SSZ)
```

Comment:
Please move this file to `shared/ssz/README.md`

Reply:
Done. Thanks